packages-ui/roosterjs-react/lib/emoji/components/EmojiIcon.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { css } from '@fluentui/react/lib/Utilities';
+import { Emoji } from '../type/Emoji';
+import { EmojiIconStyle } from '../type/EmojiStyle';
+import { getDataAndAriaProps } from '../utils/getAriaAndDataProp';
+import { mergeStyleSets } from '@fluentui/react/lib/Styling';
+import { Strings } from '../type/Strings';
+
+/**
+ * @internal
+ * Emoji icon data
+ */
+export interface EmojiIconProps {
+    id: string;
+    emoji: Emoji;
+    strings: Strings;
+    onClick?: (e: React.MouseEvent<EventTarget>) => void;
+    onMouseOver?: (e: React.MouseEvent<EventTarget>) => void;
+    onFocus?: React.FocusEventHandler<HTMLButtonElement>;
+    isSelected?: boolean;
+    emojiIconStyle: EmojiIconStyle;
+}
+
+/**
+ * @internal
+ * Emoji icon component
+ */
+export default function EmojiIcon(props: EmojiIconProps) {
+    const { emoji, onClick, isSelected, onMouseOver, emojiIconStyle, onFocus, strings, id } = props;
+    const content = strings[emoji.description];
+
+    return (
+        <button
+            id={id}
+            role="option"
+            className={css(classNames.emoji, emojiIconStyle.className, {
+                [classNames.roosterEmojiSelected]: isSelected,
+                [emojiIconStyle.selectedClassName]: isSelected,
```

Comment:
just use classNames should be good enough for now. If need we can add the support for emojiIconStyle later. Same for other customizable options.